src/app/(main)/(routes)/(public)/page.tsx

```diff
@@ -45,22 +36,22 @@ function HomeCountdown() {
         <div className="grid grid-cols-7 place-items-center items-start text-white">
           <div className="flex flex-col items-center">
             <p className="text-5xl">24</p>
-            <p className="text-xl">Days</p>
+            <p className="">Days</p>
           </div>
           <p className="text-5xl">:</p>
           <div className="flex flex-col items-center">
             <p className="text-5xl">03</p>
-            <p className="text-xl">Hours</p>
+            <p className="">Hours</p>
           </div>
           <p className="text-5xl">:</p>
           <div className="flex flex-col items-center">
             <p className="text-5xl">46</p>
-            <p className="text-xl">Minutes</p>
+            <p className="">Minutes</p>
           </div>
           <p className="text-5xl">:</p>
           <div className="flex flex-col items-center">
             <p className="text-5xl">02</p>
-            <p className="text-xl">Seconds</p>
+            <p className="">Seconds</p>
```

Comment:
Why have empty className tag? Just have no styling if there's no styling to the specific element at hand

Reply:
Used to have styling, then removed. Accidentally kept them after highlighting all _text-xl_ at once. This was a mistake.